big-practice/src/js/pages/dashboard.page.js

```diff
@@ -1,114 +1,115 @@
-import { Validation } from '../validates/form.validate';
-import { EMPTY_TEXT, MOVIES_API, MESSAGES } from '../constants/message.constant';
+import { formValidate } from '../validates/form.validate';
+import { MOVIES_API } from '../constants/url-api.constant';
+import { EMPTY_TEXT, MESSAGES } from '../constants/message.constant';
 import {
-  getApi, postApi, putApi, deleteApi,
+  get, post, put, remove,
```

Comment:
ApiService is class, why you need to import get/post/put/remove?

Reply:
Updated!